activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -58,6 +58,15 @@ def custom_primary_key?
         options[:primary_key] != default_primary_key
       end
 
+      def validate
+        options.fetch(:validate, true)
+      end
```

Comment:
This is a little strange to me because you could pass in `validate: nil` or `validate: 'foo'` and hence you wouldn't be getting back a boolean from this method.